applications/salk-portal/src/components/ExperimentSidebar.jsx

```diff
@@ -151,8 +159,8 @@ const ExperimentSidebar = () => {
 
   const PopulationLabel = ({labelText, dotColor}) => {
     return (
-      <Typography variant='span' className='population-label-container' >
-       <Box className='population-label-dot' variant='span' style={{backgroundColor:dotColor}}/>
+      <Typography className='population-label-container' >
+       <Box className='population-label-dot' style={{backgroundColor:dotColor}}/>
        <Typography className='population-label-text' variant='p'>{labelText}</Typography>
```

Comment:
@paularah an extra Typography element is not required for the text. Please note we are already inside a Typography. You can just use the parent Typography and add you styles in that class for the text and structure  and the just another class for dot, this way you do not need three separate classes.